src/cascadia/TerminalAzBridge/TerminalAzBridge.vcxproj

```diff
@@ -11,6 +11,7 @@
     <OpenConsoleUniversalApp>false</OpenConsoleUniversalApp>
     <ApplicationType>Windows Store</ApplicationType>
     <NoOutputRedirection>true</NoOutputRedirection>
+    <TargetPlatformIdentifier>Windows</TargetPlatformIdentifier>

```

Comment:
`<!-- LOAD BEARING CODE: If we don't say Windows here, the project system might get confused and start building and not building APPXs because of the implicit decisions on UWP. -->`

Reply:
idk about that one -- this feels more _correct_ (like, just that we shoulda specified it).. but i can see the argument